src/database/queries/mentorExtension.js

```diff
@@ -310,4 +310,28 @@ module.exports = class MentorExtensionQueries {
 			type: Sequelize.QueryTypes.UPDATE,
 		})
 	}
+
+	static async getMentorsExtensions(userIds, attributes = []) {
+		try {
+			const queryOptions = { where: { user_id: { [Op.in]: userIds } }, raw: true }
+			if (attributes.length > 0) {
+				queryOptions.attributes = attributes
+			}
+			const mentors = await MentorExtension.findAll(queryOptions)
+			const mentorMap = new Map(mentors.map((mentor) => [mentor.user_id, mentor]))
+			const validMentors = []
+			const invalidMentors = []
+			userIds.forEach((userId) => {
+				const mentor = mentorMap.get(userId)
+				if (mentor) {
+					validMentors.push(mentor)
+				} else {
+					invalidMentors.push(userId)
+				}
+			})
```

Comment:
We cannot have these here. Functions inside queriers should return DB response as it is.

Reply:
ok can i use it in create session as a function ? @nevil-mathew

Reply:
You can move this to the appropriate function, but this has to be moved from queries.